Sources/Penny/Handlers/CoinFinder.swift

```diff
@@ -1,4 +1,4 @@
-import Foundation
+import Foundation /// Need to import the whole thing for the `String.split()` function call below?
```

Comment:
It should be in essentials unless it relies on ICU to ensure unicode is split correctly but I don't think that's the case, I thought that was all reimplemented in Swift for this reason

Reply:
Not sure what exactly is going on but things don't compile without importing the whole thing.
Something about `CharacterSet` I think, for one.

Reply:
Ah right IIRC `CharacterSet` is still not ported over? There were some issues around it in the main OpenAPIGenerator repo.

Reply:
This is my own comment from a few months ago. Apparently it was accurate:
https://github.com/apple/swift-openapi-runtime/pull/107#issuecomment-2181552319

Reply:
Yes, the problem was `CharacterSet`: https://github.com/vapor/penny-bot/pull/253/commits/e96f9918926531791ec2d870d65c5a1deedc02b5